.github/workflows/check-patch.yaml

```diff
@@ -0,0 +1,19 @@
+name: Lint Python
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install requirements
+      run: bash ./scripts/ci/install.sh
```

Comment:
`pylint` requires all of the dependencies installed to check for `import-error`. Could either disable it (which is bad) or, need to install all of the dependency before-hand.